qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -1062,3 +1064,35 @@ def from_symplectic(cls, z, x, phase=0):
         """
         base_z, base_x, base_phase = cls._from_array(z, x, phase)
         return cls(BasePauli(base_z, base_x, base_phase))
+
+    def _qw_anticommutation_graph(self):
+        """Create qubit wise anti commutation graph with edges (i, j) if i and j are not commutable.
```

Comment:
I think we should somehow be explicit that we are constructing an edge list here. Maybe something like "Create an edge list representing the qubit-wise anti-commutation graph. An edge is present if ..."
(The wording here follows the opflow code, I think)